python/pyspark/sql/streaming/stateful_processor.py

```diff
@@ -359,6 +359,14 @@ class StatefulProcessor(ABC):
     Class that represents the arbitrary stateful logic that needs to be provided by the user to
     perform stateful manipulations on keyed streams.
 
+    NOTE: The type of input data and return are different by which method is called, following:
```

Comment:
nit: `type of input data and return type are different based on which method is called, such as`

Reply:
Did we update these or you prefer to retain what you have ?